py3status/parse_config.py

```diff
@@ -292,6 +310,41 @@ def make_value(self, value):
             return None
         return value
 
+    def make_value_from_env(self, value):
+        # Extract the environment variable and type
+        match = re.match('env\(([0-9A-Z_]+)(, *[a-z]+)?\)', value)
```

Comment:
the `(, *[a-z]+)` should be `(,\s?*[a-z]+)` because there's a high chance that they forget the space

Reply:
erm; ` *` matches zero-or-more spaces; `\s?*` at least to me looks like a syntax error (and the [docs](https://docs.python.org/3/library/re.html#regular-expression-syntax) seem to think so as well)

Reply:
`'env\(\s*([0-9A-Z_]+)(,\s*[a-z]+)?\s*\)'`

Let's just not be strict with the whitespace.  Also we are being case sensitive is there a reason we are doing this?

Reply:
You're right @girst I was sloppy and missed the *

I agree with @tobes proposal

Reply:
No reason for the lowercase, I threw a `string.lower()` call in there to be compatible. Thanks for the updated regex! I added support for mixed case and underscores as well.